packages/govuk-frontend/src/govuk/components/character-count/character-count.mjs

```diff
@@ -132,7 +132,7 @@ export class CharacterCount extends GOVUKFrontendComponent {
     })
 
     // Determine the limit attribute (characters or words)
-    this.maxLength = this.config.maxwords || this.config.maxlength
+    this.maxLength = this.config.maxwords || this.config.maxlength || Infinity
```

Comment:
**question** Given how we validate the configuration, we'll never reach the `|| Infinity`, I think. What's requiring us to keep it?

Reply:
Ah it's only moved it from the class field to here to make the compiler happy in strict mode

Perhaps when we pick up https://github.com/alphagov/govuk-frontend/issues/4230 we can look at `CharacterCountConfig` becoming `Required<CharacterCountConfig>` when all fields are validated?

Then the compiler will know a fallback is no longer needed